.githooks/pre-commit

```diff
@@ -10,7 +10,8 @@ done
 
 if [[ -n "${STAGED_GO_FILES}" ]]; then
     set -o errexit
-    go test -race ./internal...
+    make test
+    make integration-test
```

Comment:
Probably might not want these here in the future, but I think for now its ok

Reply:
works for me but how do you feel creating a `unit-test` target and making `make test` call both unit and integration?

Reply:
👍  Im ok with that